pip/pep425tags.py

```diff
@@ -124,7 +124,54 @@ def get_platform():
         split_ver = release.split('.')
         return 'macosx_{0}_{1}_{2}'.format(split_ver[0], split_ver[1], machine)
     # XXX remove distutils dependency
-    return distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    result = distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    if result == "linux_x86_64" and sys.maxsize == 2147483647:
```

Comment:
Yeah, this fixes the same bug that PR #1377 was trying to address; [see also this mailing list message](https://mail.python.org/pipermail/distutils-sig/2016-January/028215.html). The bug has been around a long time, but it becomes much more urgent to fix as soon manylinux support lands; in the past it only caused a problem if you specifically did `pip install -f <directory with 64 bit wheels>` from a 32-bit userland, but once 64-bit linux wheels are on pypi it will become possible to hit by just doing `pip install`. So it's important to land this fix at the same time we enable manylinux support.